tests/app/main/test_suppliers.py

```diff
@@ -367,7 +367,7 @@ def test_shows_gcloud_7_closed_message_if_pending_and_application_done(
     @mock.patch("app.main.views.suppliers.data_api_client")
```

Comment:
The below test name was declared twice meaning this test was run but the one of the same name above wasnt.